src/main/java/net/mcreator/element/types/Block.java

```diff
@@ -58,6 +58,7 @@
 
 	public String itemTexture;
 	public String particleTexture;
+	public String sensitivity;

```

Comment:
For now consider making it so if the selected material is iron, make it everything, otherwise mobs. Doors do the same with redstone control.

If we added all possible fields for all tx blocks things would quickly get messy, so in the future, a proper system for tx block parameters needs to be implemented, but for now, we should keep tx blocks with default properties.

Reply:
Everything is used by wooden pressure plates. Iron, stone and I think gold pressure plates are only activated with mobs.

Reply:
Ok then use check for these materials.